scripts/install_quantization_libs.py

```diff
@@ -1,29 +1,11 @@
 import argparse
 import os
-import re
 import subprocess
 import sys
 
 EXTERNAL_REPOS_DIR = "external_repos"
 
 
-def process_setup_file(setup_file_path):
```

Comment:
No needed anymore, as the dependency list for autoawq is now a `>=` and not a `==` so this is not needed